CHANGELOG.md

```diff
@@ -1,4 +1,23 @@
 ## 3.16.0 (Unreleased)
+FEATURES:
+* **New Data Source:** monitoring: `google_monitoring_uptime_check_ips` ([#1912](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1912))
+* **New Resource:** firebase: Added new `firebase_project_location` resource to finalize the firebase location. ([#1919](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1919))
```

Comment:
```suggestion
* **New Resource:** `firebase_project_location`: finalizes the firebase location. ([#1919](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1919))
```

something like this? we generally just put the name of the new resource here, but I suppose the description can't hurt too